variables.tf

```diff
@@ -24,46 +24,11 @@ variable "skip_region_validation" {
 ################################
 # RAM user
 ################################
-variable "create" {
```

Comment:
I think the `create` should be kept and it can used to control whether create users.

Reply:
if the user does not want to create users, commenting the variable "users" will control whether to create users